delta/plugins/composite-views/src/main/resources/composite-views.conf

```diff
@@ -3,6 +3,13 @@ plugins.composite-views {
   enabled = true
   # the priority of the plugin
   priority = 5
+  # blazgraph instance for composite views
+  # by default it is the same as the one defined in the blazegraph plugin
+  blazegraph = {
```

Comment:
Another possibility is to group `base`, `credentials`, and `query-timeout` at blazegraph plugin level into a single object.